threedigrid_builder/grid/boundary_conditions.py

```diff
@@ -266,8 +266,8 @@ def is_groundwater(kcu: LineType) -> bool:
         }
 
     @staticmethod
-    def boundary_type_is_groundwater(boundary_type: BoundaryType) -> bool:
-        return boundary_type in {
+    def type_is_groundwater(type: BoundaryType) -> bool:
```

Comment:
The aim of this migration is to minimize any effects outside of the 4 repos involved. So it's better to not change function names and change variable names as sparsely as possible. 

```suggestion
    def boundary_type_is_groundwater(type: BoundaryType) -> bool:
```